ArmaReforgerServerTool/Forms/Main.cs

```diff
@@ -793,18 +793,19 @@ private void CreateServerParameterControls()
                 ParameterFriendlyName = "Rcon Max Clients",
                 ParameterTooltip = Constants.SERVER_PARAM_RCON_MAX_CLIENTS_TOOLTIP_STR,
                 ParameterIncrement = 1,
-                ParameterMin = 1,
-                ParameterMax = 16,
-                ParameterValue = 16
+                ParameterMin = Rcon.MIN_CLIENTS,
+                ParameterMax = Rcon.MAX_CLIENTS,
+                ParameterValue = Rcon.DEFAULT_CLIENTS
             };
             serverParameters.Controls.Add(rconMaxClients);
             ServerParameterSelect rconPermission = new()
             {
                 ParameterName = "rconPermission",
                 ParameterFriendlyName = "Rcon Permission",
                 ParameterTooltip = Constants.SERVER_PARAM_RCON_PERMISSION_TOOLTIP_STR,
-                ParameterValue = new[]{"admin", "monitor"}
+                ParameterValue = Rcon.PERMISSIONS,
             };
+            rconPermission.ParameterValueSelection(Rcon.PERMISSIONS[(int)Rcon.DEFAULT_PERMISSION]);
```

Comment:
Instead of this, in the Rcon struct you can define Rcon.DEFAULT_PERMISSION as `Rcon.PERMISSIONS[0]`. It will be a bit cleaner. I don't like seeing casts if it can be avoided.